src/actions.cpp

```diff
@@ -210,67 +211,37 @@ bool Actions::registerEvent(Event_ptr event, const pugi::xml_node& node)
 bool Actions::registerLuaEvent(Action* event)
 {
 	Action_ptr action{ event };
-	if (action->getItemIdRange().size() > 0) {
-		if (action->getItemIdRange().size() == 1) {
-			auto id = action->getItemIdRange().front();
-			auto result = useItemMap.emplace(id, std::move(*action));
+	if (!action->getItemIdRange().empty()) {
+		const auto& v = action->getItemIdRange();
+		for (auto id : v) {
```

Comment:
what if we used better names instead of "v"?

Reply:
Absolutely. Renamed to `range`